transformations/character_duplication/test.json

```diff
@@ -0,0 +1,50 @@
+{
+  "type": "character_duplication",
+  "test_cases": [
+    {
+      "class": "CharacterDuplication",
+      "inputs": {
+        "sentence": "Andrew finally returned the French book to Chris that I bought last week"
+      },
+      "outputs": [{
+        "sentence": "Anndrew ffinnallly returrned thee  French book too Chhris that I bought last  week"
+      }]
+    },
+    {
+      "class": "CharacterDuplication",
+      "inputs": {
+        "sentence": "Sentences with gapping, such as Paul likes coffee and Mary tea, lack an overt predicate to indicate the relation between two or more arguments."
+      },
+      "outputs": [{
+        "sentence": "Seentencees  witth gappiing, succhh as Paul likess cooffee and Mary tea, lackk an overt predicate ttoo indiicate tthe relation between two  orr moree arrguuments."
+      }]
+    },
+    {
+      "class": "CharacterDuplication",
+      "inputs": {
+        "sentence": "Alice in Wonderland is a 2010 American live-action/animated dark fantasy adventure film"
+      },
+      "outputs": [{
+        "sentence": "Allice inn WWondderland  is a 200110 American livve-aaction/animated dark fanntasy adventure film"
```

Comment:
The same is for the double letter in the beginning or a 6-figure number, which should represent a year. Please consider adding some rules to change that behaviour.

Reply:
For the same reason as before, I disagree about the double letter in the beginning, but I agree with you about not duplicating digits. I have added a rule to exclude digits from duplication in eb09bbcab3d2d47742ef47e353ec315e3ab05627. Thank you for the suggestion